### ISSUE TITLE
[Blocking BUG] .CSV importing problem (cont of #1107)

### ISSUE THREAD

**Issue by kurde78** [jpatokal/openflights#1108]
Uploading the file works now after #1107 was fixed, but importing still does not. When I try with the sample 5 entries .csv from your CSV Format Spec example, I only get info about parsing and the first line displayed. No button to confirm, flights not imported...

OpenFlights: Import
Upload successful. Parsing...

Results
Tmpfile phpbYrftn
ID Date Flight From To Miles Time Plane Reg Seat Class Type Reason Trip Comment
1 2008-01-18 10:30:10 ID 4533 SV559 ID 2188 ID 2082 542 01:35 ID 3 HZ-AKH 40C Aisle Economy Business 16 First, to Saudi!

**Comment by Adam-C-Smith** [comment]
Same problem here.

**Comment by benmacd80** [comment]
Same issue with me. I edited my openflights backup and split it into years. 2022 was able to import just fine, 2021, 2020 etc I get the parsing and it displays the flights but no button to actually click to import.

**Comment by kurde78** [comment]
Ok, after 8 days waiting for a showstopper to be at least assigned to someone, I got an impression this project is dead. 

Moving to flightradar24 I just discovered to have a beautiful (and working) personal flight keeping log. With working import too :)

**Comment by wshaw17** [comment]
I've been a big fan of OF but this is serious bug and agree with kurde78 people are moving on to other formats. Such a shame…this WAS a great tool.

**Comment by jpatokal** [comment]
New airport edit suggestion submitted by ordtpe:

Array
(
    [apid] => 7803
    [name] => Mannheim HBF
    [city] => Mannheim
    [country] => Germany
    [iata] => 
    [icao] => 
    [x] => 8.469722
    [y] => 49.479722
    [elevation] => 1000
    [timezone] => 1
    [dst] => E
)
;

Existing, potentially conflicting airport information:

```
Array
(
    [name] => Mannheim HBF
    [city] => Mannheim
    [country] => Germany
    [country_code] => 
    [iata] => 
    [icao] => 
    [x] => 8.469858
    [y] => 49.471
    [elevation] => 1000
    [apid] => 7803
    [uid] => 5693
    [timezone] => 1
    [dst] => E
    [tz_id] => Europe/Berlin
    [type] => station
    [source] => User
)

```

Cross-check this edit on other sites with compatible licensing:
- OurAirports: http://ourairports.com/airports//pilot-info.html
- Wikipedia: http://www.google.com/search?q=wikipedia%20%20airport&btnI

**Comment by jpatokal** [comment]
Apologies for the delay, this is likely a duplicate of #1116 and as such should be fixed now.  Please reopen if still seeing problems, and include the `Tmpfile` value so I can replicate.

**Comment by wshaw17** [comment]
@jpatokal thanks—it's working now!
